go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/CosmWasm/wasmd
 go 1.15
 
 require (
-	github.com/CosmWasm/wasmvm v0.14.0-beta1
+	github.com/CosmWasm/wasmvm v0.14.0-beta3
```

Comment:
There is a reference in the [Dockerfile](https://github.com/CosmWasm/wasmd/blob/master/Dockerfile#L18) that we need to update. 🕵️

Reply:
Will do that and `go mod tidy`